src/index.js

```diff
@@ -22,41 +23,24 @@ if (!isProduction) {
 
 // used in testing heroku
 app.get('/', (req, res) => res.status(200).json({
-  status: 200,
   message: 'Welcome to Barefoot Nomad'
 }));
 
-app.use(router);
-app.use((req, res, next) => {
-  const err = new Error('Not Found');
-  err.status = 404;
-  next(err);
+// to throw an error to the logger
+app.get('/err', (req, res) => {
+  throw new ErrorHandler('Error here');
 });
 
-
-if (!isProduction) {
-  app.use((err, req, res, next) => {
-    console.log(err.stack);
-
-    res.status(err.status || 500);
-
-    res.json({
-      errors: {
-        message: err.message,
-        error: err
-      }
-    });
-  });
-}
+app.use(router);
+app.use((req, res, next) => res.status(404).json({
+  error: 'Not found'
+}));
 
 
 app.use((err, req, res, next) => {
```

Comment:
This is nice, this means all errors would pass through express middleware. Kindly have a look at this implementation https://github.com/timolinn/node-starter/blob/13a9dfffba5d0565abdf91c55a44de43be90e259/src/app.js#L45.
There are two error handling middlewares there, for `production` and `development`.